Userland/Utilities/wasm.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>
+ * Copyright (c) 2022, the SerenityOS developers.
```

Comment:
Removing someone's name from the copyright header isn't very nice. The following would be better:
```
 * Copyright (c) 2021-2022, Ali Mohammad Pur <mpfard@serenityos.org>
 * Copyright (c) 2022, the SerenityOS developers.
```

Reply:
Fixed

Reply:
Why are we changing the date range of @alimpfard's copyright?

Reply:
Just keep `Copyright (c) 2021, Ali Mohammad Pur <mpfard@serenityos.org>` ?